renderer/reducers/payment.js

```diff
@@ -427,8 +402,4 @@ export { paymentSelectors }
  * @param  {object} action Action
  * @returns {object} Next state
```

Comment:
Same here. docblock can probably go

Reply:
The documented  params are not correct here. I would probably just remove this docblock completely since we are no longer defining a method here.

Reply:
This docblock still needs to be removed @korhaliv . Same goes for `network.js`